publish.js

```diff
@@ -0,0 +1,272 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var publish = {};
+var pkg = require("./package.json");
+publish.execSync = require("child_process").execSync;
+// TODO: The supported version of node.js does not yet support ES6 template strings
+// When version node.js 4.x.x is supported this can be replaced by native support.
+var es6Template = require("es6-template-strings");
+
+var defaults = pkg.defaultOptions;
+
+// From dedupe-infusion ( https://github.com/fluid-project/dedupe-infusion )
+// Licensed under BSD-3-Clause
+publish.shallowMerge = function (target/*,  ... */) {
```

Comment:
Rather than proliferate my existing bad practice from dedupe-infusion, I suggest that you depend on this widely-used npm package instead: https://www.npmjs.com/package/extend